setup.py

```diff
@@ -11,11 +11,9 @@
 
 setup(
     name="pylint-pytest",
-    version="1.1.2",
-    author="Reverb Chu",
```

Comment:
That is your perogative @Pierre-Sassoulas.

Should we at least keep the `author's` name?
Will `pyproject.toml` allow us to have an array of names?

Reply:
Yes definitely. And we'll have to rebase #8 so we'll have to add you again.

Reply:
I meant

```py
    version="1.1.2",
    author="Reverb Chu",
    maintainer="Stavros Ntentos",
    maintainer_email="133706+stdedos@users.noreply.github.com",
```

but sure 😅

Reply:
Ha right let's fix that in 1.1.3 actual release. Do you want to release a a1 with the fix yourself ?

Reply:
Sunday is a bit crammed for me 😅 

I am only trying to unblock the work you are doing now; I should come back in the night